packages/fxa-payments-server/src/index.scss

```diff
@@ -34,7 +34,129 @@ h3.billing-title {
   }
 }
 
-#main-content {
+// starting to replace the main-content id with this class
+.main-content {
+  display: flex;
+  padding-top: 0;
+
+  .input-row {
+    // Stripe element class names
+    // https://stripe.com/docs/stripe-js/reference#the-element-container
+
+    .StripeElement {
+      @include input-element();
```

Comment:
Huh, I thought we had trouble styling the form due to the iframe. Check out what we're [doing here](https://github.com/mozilla/fxa/blob/master/packages/fxa-payments-server/src/components/PaymentForm/index.tsx#L276), at the very least we should update `SMALL_DEVICE_RULE` there. Does this mean we could potentially remove `mkStripeElementStyles`? Would need to do more digging, I'll leave this thought here though since this is a draft PR🤔

Reply:
I'd like to avoid refactors like this, maybe we can file a follow up issue. This was just moved from the parent id element to a class

Reply:
Yeah, I wasn't thinking you'd address that in this PR, but might be worth looking into as a new issue.